google/ggrc-core
# GGRC-5943 Assign null to attribute_object_id when person is deleted

# Issue description

Person LCA cannot be cleared

# Steps to test the changes

1. Create asmt with person LCA
2. Fill value of person LCA
3. Refresh page. Clear value of person LCA
Actual result: It comes back after AJAX saving. Person LCA can't be cleared
Expected result: Person LCA should be clear-able

# Solution description

Assign null to attribute_object_id when person is deleted

# Sanity checklist




# PR Review checklist
